lib/plugins/install/tests/install.js

```diff
@@ -77,5 +129,18 @@ describe('Install', () => {
         expect(downloadStub.args[0][0]).to.equal(`${install.options.url}/archive/master.zip`);
       });
     });
+
+    it('should download and rename the service based on the GitHub URL', () => {
```

Comment:
@pmuens, so basically either serverless.yml should be created to the download folder beforehand or downloadStub should pass zip containing serverless.yml as a payload. The first one might not work, because file should be created to folder before running the install function, which checks if the folder exists and if so it won't install the package. I'll check if I can pass the zip as a payload.